include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -1866,15 +1866,12 @@ class charge_distribution_surface<Lyt, false> : public Lyt
         std::sort(strg->sidb_order.begin(), strg->sidb_order.end());
         this->foreach_cell([this, &cs](const auto&) { strg->cell_charge.push_back(cs); });
 
-        assert((((this->num_cells() < 41) && (strg->simulation_parameters.base == 3)) ||
```

Comment:
@marcelwa must be discussed.

Reply:
Let's gladly discuss this later today. From first glance, I support the removal.